.github/workflows/ci.yaml

```diff
@@ -23,8 +23,8 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macOS-latest]
-        python-version: [2.7] 
-        #python-version: [2.7, 3.8] 
+        #python-version: [2.7]
+        python-version: [2.7, 3.8]
```

Comment:
Testing in python 3.8

Reply:
See https://github.com/Becksteinlab/MDPOW/pull/166#issuecomment-885357147 on setting the test matrix. Start out simple as you do (just make sure you don't run a full matrix with all GROMACS versions, OS, and Python versions — that would be unnecessary and consume to many resources and time)